validator/validators/performance/performance_validator.go

```diff
@@ -45,19 +46,34 @@ func NewPerformanceValidator(vConfig models.ValidateConfig) models.ValidatorFact
 }
 
 func (s *PerformanceValidator) CheckData(startTime, endTime time.Time) error {
-	metrics, err := s.GetPerformanceMetrics(startTime, endTime)
-	if err != nil {
-		return err
-	}
+	if strings.Contains(s.vConfig.GetTestCase(), "windows") {
```

Comment:
I would pass in the os family using the same << >> approach so we dont just rely on the names here.

Reply:
Can you explain what you mean by "<< >> approach"? I'm interested in doing this for the stress tests.

Reply:
Sure yes. Our parameters.yml files have token replacement logic ([example](https://github.com/aws/amazon-cloudwatch-agent-test/blob/ee898e8bc44f4be0a16495c752dbf7e3478526ba/test/stress/system/parameters.yml#L15)) that enables specifying placeholders in there which will be substituted with variables that can be passed in from terraform.
Hence we could use a similar pattern to pass in the os family which is already available as a terraform var when applying these stress tests.

Reply:
Implemented this in the new revision